CHANGELOG

```diff
@@ -1,3 +1,6 @@
+* Thu Jun 14 2018 Nick Miller <nick.miller@onyxpoint.com> - 7.6.0-0
+- Update systemd fixtures and CI assets
```

Comment:
This doesn't appear to update anything related to `systemd`, is something missing or is this message incorrect?